libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Operations.kt

```diff
@@ -149,8 +148,6 @@ fun <D : Operation.Data> Operation<D>.composeJsonResponse(
 
 
 @ApolloExperimental
-@Deprecated("toApolloResponse() closes the JsonReader. Use parseResponse instead", ReplaceWith("parseResponse"))
-@ApolloDeprecatedSince(ApolloDeprecatedSince.Version.v4_0_0)
```

Comment:
Maybe we could highlight it in the doc
```kotlin
/**
 * Note that this will close the JsonReader.
 */
```

we could also make it `@ApolloInternal` as a kind of "deprecation that we can still use ourselves" 😅

Reply:
> Maybe we could highlight it in the doc

I've added a comment there. The whole parsing API needs KDoc. Problem for another day.

> we could also make it @ApolloInternal as a kind of "deprecation that we can still use ourselves"

That's the thing I'm not sure I want to do. Problem for another day.